web/themes/custom/pineapple/pineapple.theme

```diff
@@ -52,11 +52,9 @@ function pineapple_theme_suggestions_input_alter(array &$suggestions, array $var
 /**
  * Implements hook_theme_suggestions_user_alter().
  */
-function pineapple_theme_suggestions_user_alter(array &$suggestions, array $variables) {
+function pineapple_theme_suggestions_user_alter(array &$suggestions, array $variables) {$entity_id = $variables["elements"]["#user"]->entityTypeId;
```

Comment:
What's this "$entity_id = $variables["elements"]["#user"]->entityTypeId;" doing here?